Many changes

TheBoi1 opened the pull request:
-New Background img
-Fixed buttons
-Added Audio
-Made Button Text Easier To Read

JustAnAsian commented:
Your contributions are unnecessary. Our fire is better than yours and your living rats don't support our ideas of dead rats. All you did was indent and added basically nothing. The things you did add were terrible ideas.

TheBoi1 commented:
I guess you didn’t listen to the sound file I added. You don’t want audio in your project? That’s fine by me, your loss.

-Armon Bhau

> On Mar 5, 2018, at 2:13 PM, Asian Boy <notifications@github.com> wrote:
> 
> Your contributions are unnecessary. Our fire is better than yours and your living rats don't support our ideas of dead rats. All you did was indent and added basically nothing. The things you did add were terrible ideas.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>
